easistrain/EDD/calibrationEDD.py

```diff
@@ -1,22 +1,12 @@
+import argparse
```

Comment:
Note that I renamed the file `calibrationEdd` to `calibrationEDD` for consistency with the other files